kr8s/_auth.py

```diff
@@ -182,10 +180,6 @@ async def _load_kubeconfig(self) -> None:
                 )
         if "token" in self._user:
             self.token = self._user["token"]
-        if "username" in self._user:
-            self.username = self._user["username"]
-        if "password" in self._user:
-            self.password = self._user["password"]
```

Comment:
I wonder if we should raise an exception if these are set in the config to say "username/password authentication was removed in Kubernetes 1.19, kr8s doesn't not support this Kubernetes version".

Reply:
will do this, good!